conn/physic/units.go

```diff
@@ -352,6 +352,114 @@ const (
 	Fahrenheit      Temperature = 555555555 * NanoKelvin
 )
 
+// Power is a measurement of power stored as a nano watts.
+//
+// The highest representable value is 9.2GW.
+type Power int64
+
+// String returns the power formatted as a string in watts.
+func (p Power) String() string {
+	return nanoAsString(int64(p)) + "W"
+}
+
+const (
+	// Watt is unit of power J/s, kg⋅m²⋅s⁻³
+	NanoWatt  Power = 1
+	MicroWatt Power = 1000 * NanoWatt
+	MilliWatt Power = 1000 * MicroWatt
+	Watt      Power = 1000 * MilliWatt
+	KiloWatt  Power = 1000 * Watt
+	MegaWatt  Power = 1000 * KiloWatt
+	GigaWatt  Power = 1000 * MegaWatt
+)
+
+// Energy is a measurement of work stored as a nano joules.
+//
+// The highest representable value is 9.2GJ.
+type Energy int64
+
+// String returns the energy formatted as a string in Joules.
+func (e Energy) String() string {
+	return nanoAsString(int64(e)) + "J"
+}
+
+const (
+	// Joule is a unit of work. kg⋅m²⋅s⁻²
+	NanoJoule  Energy = 1
+	MicroJoule Energy = 1000 * NanoJoule
+	MilliJoule Energy = 1000 * MicroJoule
+	Joule      Energy = 1000 * MilliJoule
+	KiloJoule  Energy = 1000 * Joule
+	MegaJoule  Energy = 1000 * KiloJoule
+	GigaJoule  Energy = 1000 * MegaJoule
+)
+
+// ElectricalCapacitance is a measurement of capacitance stored as a pico farad.
+//
+// The highest representable value is 9.2MF.
+type ElectricalCapacitance int64
+
+// String returns the energy formatted as a string in Farad.
+func (c ElectricalCapacitance) String() string {
+	return picoAsString(int64(c)) + "F"
+}
+
+const (
+	// Farad is a unit of capacitance. kg⁻¹⋅m⁻²⋅s⁴A²
+	PicoFarad  ElectricalCapacitance = 1
+	NanoFarad  ElectricalCapacitance = 1000 * PicoFarad
+	MicroFarad ElectricalCapacitance = 1000 * NanoFarad
+	MilliFarad ElectricalCapacitance = 1000 * MicroFarad
+	Farad      ElectricalCapacitance = 1000 * MilliFarad
+	KiloFarad  ElectricalCapacitance = 1000 * Farad
+	MegaFarad  ElectricalCapacitance = 1000 * KiloFarad
+)
+
+// LuminousIntensity is a measurement of wavelength power stored as a
+// nano candela.
+//
+// This is one of the base unit in the International System of Units.
+//
+// The highest representable value is 9.2Gcd.
+type LuminousIntensity int64
+
+// String returns the energy formatted as a string in Candela.
+func (l LuminousIntensity) String() string {
+	return nanoAsString(int64(l)) + "cd"
+}
+
+const (
+	// Candela is a unit of luminous intensity. cd
+	NanoCandela  LuminousIntensity = 1
+	MicroCandela LuminousIntensity = 1000 * NanoCandela
+	MilliCandela LuminousIntensity = 1000 * MicroCandela
+	Candela      LuminousIntensity = 1000 * MilliCandela
+	KiloCandela  LuminousIntensity = 1000 * Candela
+	MegaCandela  LuminousIntensity = 1000 * KiloCandela
+	GigaCandela  LuminousIntensity = 1000 * MegaCandela
+)
+
+// LuminousFlux is a measurement of luminous flux stored as a nano lumens.
```

Comment:
".. a measurement of the total quantity of visible light emitted by a source, stored as ..."